apps/design/backend/schema.sql

```diff
@@ -5,4 +5,14 @@ create table elections (
   precinct_data text not null,
   layout_options_data text not null,
   created_at timestamp not null default current_timestamp
-);
\ No newline at end of file
+);
+
+create table background_tasks (
+  id text primary key,
+  task_name text not null,
+  payload text not null,
+  created_at timestamp not null default current_timestamp,
+  started_at timestamp,
+  completed_at timestamp,
+  error text
+);
```

Comment:
As noted in the PR description, I decided to mimic the DB schema that Arlo uses for background task processing: https://github.com/votingworks/arlo/blob/7a2689cf938efcf8e2733e2d8732eadfc020be48/server/models.py#L977-L988

I didn't include fields for progress tracking yet. Can add those when the need arises.

I also considered foregoing `task_name` and incorporating it into the `payload`, but decided to keep the separate field since being able to query on specific `task_name` values feels useful.